rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -800,6 +801,37 @@ int LuaSyncedCtrl::SetAlly(lua_State* L)
 }
 
 
+/*** Changes the start box position of an allyTeam.
+ *
+ * @function Spring.SetAllyTeamStartBox
+ * @number allyTeamID
+ * @number xMin left start box boundary (elmos)
+ * @number zMin top start box boundary (elmos)
+ * @number xMax right start box boundary (elmos)
+ * @number zMax bottom start box boundary (elmos)
+ * @treturn nil
+ */
+int LuaSyncedCtrl::SetAllyTeamStartBox(lua_State* L)
+{
+	const unsigned int allyTeamID = luaL_checkint(L, 1);
+	const float xMin = luaL_checkfloat(L, 2);
+	const float zMin = luaL_checkfloat(L, 3);
+	const float xMax = luaL_checkfloat(L, 4);
+	const float zMax = luaL_checkfloat(L, 5);
+
+	if (!teamHandler.IsValidAllyTeam(allyTeamID))
+		return 0;
+
+	const float startRectLeft = xMin / (mapDims.mapx * SQUARE_SIZE);
+	const float startRectTop = zMin / (mapDims.mapy * SQUARE_SIZE);
+	const float startRectRight = xMax / (mapDims.mapx * SQUARE_SIZE);
+	const float startRectBottom = zMax / (mapDims.mapy * SQUARE_SIZE);
```

Comment:
since i'm already nitpicking:
```suggestion
	const float startRectLeft   = xMin / (mapDims.mapx * SQUARE_SIZE);
	const float startRectTop    = zMin / (mapDims.mapy * SQUARE_SIZE);
	const float startRectRight  = xMax / (mapDims.mapx * SQUARE_SIZE);
	const float startRectBottom = zMax / (mapDims.mapy * SQUARE_SIZE);
```

Reply:
Done.